Parsers/ASimNetworkSession/ProductParsers/vimNetworkSessionMicrosoft365Defender.yaml

```diff
@@ -88,6 +92,7 @@ ParserQuery: |
               ) 
            and (array_length(httpuseragent_has_any) ==0) // if filtering by ua - return nothing
            and (array_length(dvcaction)==0 )     /// if filtered by action return nothing
+           and (eventresult=='*' or EventResult==eventresult)
```

Comment:
EventResult is not available at this point. I would calc it at the end of pre-filtering and then filter on it. Sometimes it would make sense to check on the original value. Not here I believe.